### DISCUSSION TITLE
Is there a way to create cloud-scopes/cloud-groups (UI) using the API

### DISCUSSION THREAD

**Discussion by arne21a** [CrowdStrike/falconpy#1158] [Q&A] [answered]
**Documentation page topic**
[Documentation page topic or source code module where the issue was identified.
](https://www.falconpy.io/Service-Collections/CSPM-Registration.html)

**Describe the error or typo**
We are trying to automatically onboard new azure subscriptions and aws accounts to crowdstrike cspm using the api.
As every workload has 4 accounts/subscriptions, we would like to group these in cloud-scopes (internal API Name)/cloud -groups (UI Name).
I only found the internal API endpoint the web-ui uses: `/api2/cspmregistration/classification/entities/cloud-scopes/v1`. But this api does not seem to be accessible using the generated API credentials.

Question:
Is there a way to create cloud groups using the API?

**Comment by jshcodes** [comment, marked as answer]
Hi @arne21a -

This operation is not currently available via the public OAuth2 API.

You can submit a request for this to be added via the ideas portal link available from the Falcon console.
![image](https://github.com/CrowdStrike/falconpy/assets/74007258/46723e16-ae2f-4568-93f0-9a8f01562680)

Thank you for the question!
